integration/integration_test.go

```diff
@@ -2231,10 +2215,30 @@ func trustedClusters(t *testing.T, suite *integrationTestSuite, test trustedClus
 	require.NoError(t, aux.StopAll())
 }
 
-func checkGetClusters(t *testing.T, tun reversetunnel.Server) []reversetunnel.RemoteSite {
-	clusters, err := tun.GetSites()
-	require.NoError(t, err)
-	return clusters
+func waitForClusters(t *testing.T, tun reversetunnel.Server, expected int, timeout time.Duration) {
+	tick := time.NewTicker(1 * time.Second)
```

Comment:
Add `t.Helper()` here.

Since `waitForClusters` is called in many places, we want the line number of the failure to be reported as the call site of `waitForClusters`.